src/frontends/ir/tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+set(TARGET_NAME ov_ir_frontend_tests)
```

Comment:
Please add copyrights.
You can also copy it from the file: https://github.com/openvinotoolkit/openvino/pull/12925/files#diff-ac119fdccb12d245bceebc8c0bde16cfd908fbbcd1fd9b1e0951579e112b2c80R1

Reply:
done